core/src/cost_update_service.rs

```diff
@@ -157,8 +157,12 @@ impl CostUpdateService {
     ) -> u64 {
         let mut update_count = 0_u64;
         for (program_id, program_timings) in &mut execute_timings.details.per_program_timings {
-            let current_estimated_program_cost =
-                cost_model.read().unwrap().find_instruction_cost(program_id);
+            let current_estimated_program_cost = {
+                match cost_model.read().unwrap().find_instruction_cost_raw(program_id) {
+                    Some((_, ema, _)) => ema.ceil() as u64,
+                    None => 200_000u64,
```

Comment:
can we use a constant here? I.e. the block compute limit

Reply:
I think it needs to be responsive to the tx-wide-compute-limit feature being enabled as well